wavelink/ext/spotify/__init__.py

```diff
@@ -275,10 +275,10 @@ async def search(
         if node is None:
             node: Node = NodePool.get_connected_node()
 
-        if type == SpotifySearchType.track:
-            tracks = await node._spotify._search(query=query, type=type)
-
-            return tracks[0] if return_first else tracks
+        # if type == SpotifySearchType.track:
```

Comment:
Disabling this code snippet will silence IndexError from being raised and it will return exactly the same thing as before.